test/hooks/dsl_test.exs

```diff
@@ -0,0 +1,3 @@
+defmodule Kitto.Hooks.DSLTest do
```

Comment:
Whoops. I should actually remove it. It was a test case, but there's nothing in the DSL that isn't being tested elsewhere.